src/extractLoader.js

```diff
@@ -106,6 +106,13 @@ function loadModule(request) {
  * @returns {string}
  */
 function runModule(src, filename, publicPath = "") {
+    // Rudimentary "transpilation" of ES 2015 modules (e.g file-loader 1.0.0+)
+    // TODO: Find a better way to "execute" ES 2015 modules in node context
+    // "Transpile" imports
+    src = src.replace(/^import\s+(.+)\s+from\s+([^;]+);?$/, "var $1 = require($2);");
+    // "Transpile" default export (other exports are not supported at the moment)
+    src = src.replace(/^\s*export\s+default\s*/, "module.exports = ");
```

Comment:
I agree in general. On the other hand it might not be clear to users of extract-loader, that you need babel in order to use file-loader 1.0.0+. Imho we should at least document that problem in the readme or make the "transpilation" in this PR more specific to the file-loader. What do you think?